addons/editor-square-inputs/addon.json

```
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Square block text and color inputs",
```

I don't think colour inputs needs to be in the addon name.
```suggestion
  "name": "Square block text inputs",
```